src/interfaces/EditInterface/useSaveInterface.test.js

```diff
@@ -0,0 +1,108 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+import useSaveInterface from './useSaveInterface';
+
+const username = 'test';
+const password = 'test';
+const values = { name: 'new interface' };
+const org = { id: 'orgId', interfaces: [] };
+const creds = {};
+
+describe('test useSaveInterface', () => {
+  let vendorInterface;
+  let interfaceCredentials;
+  let interfaceOrg;
+  let interfaceId;
+
+  beforeEach(() => {
+    vendorInterface = {
+      POST: jest.fn(() => Promise.resolve({ id: 'interfaceId' })),
+      PUT: jest.fn(() => Promise.resolve({ id: 'interfaceId' })),
+    };
+    interfaceCredentials = {
+      POST: jest.fn(() => Promise.resolve()),
+      PUT: jest.fn(() => Promise.resolve()),
+    };
+    interfaceOrg = {
+      PUT: jest.fn(() => Promise.resolve()),
+    };
+    interfaceId = {
+      replace: jest.fn(() => { }),
+    };
+  });
+
+  const { result } = renderHook(() => useSaveInterface());
+
+  describe('new interface, error while saving org', () => {
+    it('save creds should be called', async () => {
+      interfaceOrg = { PUT: jest.fn(() => Promise.reject()) };
+      await act(async () => {
```

Comment:
Is `act()` necessary here?
There's no state update after all.

Reply:
removed since it's not a hook anymore